### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by backwind1233**

### CODE REVIEW

**Review by yiliuTo — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -10,9 +10,9 @@
 
 - [azure-active-directory-b2c-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory-b2c)
 - [azure-active-directory-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-active-directory) 
-- [azure-cosmosdb-spring-boot-starter](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-cosmosdb)
+- [azure-spring-boot-starter-cosmos](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/spring/azure-spring-boot-starter-cosmos)
```
Comment (chenrujun) [inline README.md:13]: Do we need to use tag instead of main branch?

Comment (backwind1233) [reply README.md:13]: the link in main branch is blank,removed in main branch now

Comment (backwind1233) [reply README.md:13]: Can you merge this PR？@chenrujun

**Review by saragluna — APPROVED** [review]
LGTM.

**Pull request merged**